docs/network protocol.md

```diff
@@ -755,6 +755,43 @@ GameData is a **dict** but contains these keys and values. It's broken out into
 | location_name_to_id | dict[str, int] | Mapping of all location names to their respective ID.                                                                         |
 | checksum            | str            | A checksum hash of this game's data.                                                                                          |
 
+
+### Data Package Shared Cache
+The shared cache location is `user_cache_dir/datapackage`. `user_cache_dir` varies by platform. The
+platformdirs library is used in python to pick the correct location https://github.com/tox-dev/platformdirs.
+Similar libraries exist for other languages.
+
+Python example:
+```
+import platformdirs
+game_name = "Subnautica"
+cache_path = platformdirs.user_cache_dir("Archipelago", False)
+datapackage_path = os.path.join(cache_path, datapackage, game_name)
+```
+If your alternative library doesn't automatically include `/Archipelago/Cache/` you may need to
+assemble more of the path yourself.
+```
+DataPackageCacheFolder = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), 
+                Path.Combine("Archipelago", Path.Combine("Cache", "datapackage")));
+```
+
+Each game has its own subfolder inside the previously mentioned datapackage folder i.e.
+`user_cache_dir/datapackage/A Hat in Time`.
+
+When reading from or writing to the data package shared cache, make sure your file paths are safe, meaning they do not have
+characters that are forbidden in file names `<>:"/\|?*`.
```

Comment:
This is ambiguous. The implementation needs to strip those characters for the canonical name.